bears/generate_package.py

```diff
@@ -37,19 +41,24 @@ def create_file_from_template(template_file, output_file, substitution_dict):
         output_handle.write(template)
 
 
-def create_file_structure_for_packages(root_folder, file_to_copy, object_name):
+def create_file_structure_for_packages(root_folder, file_to_copy,
+                                       object_name, package_type):
```

Comment:
both is ok, I and most people seem to prefer own line per arg here. Do as you like it.